src/sql/workbench/browser/modal/media/modal.css

```diff
@@ -346,3 +346,25 @@
 .modal-dialog form {
 	height: 100%;
 }
+
+.modal.fade.flyout-dialog[aria-label*="Database Properties"] {
```

Comment:
Is there a class-based selector you could do here instead? The aria-label is localized, so this style would only apply to an English language mode

Reply:
Actually, let me double check on the label thing. The search I used only found a hit for Database Properties in Core. Regardless, using a custom class for the dialogs would allow us to neatly apply this style across all of them. Are we able to specify custom classes with the model view dialogs?

Reply:
Yes, the title is getting localized. https://github.com/microsoft/azuredatastudio/blob/main/extensions/mssql/src/objectManagement/localizedConstants.ts#L187

Reply:
ideally, there shouldn't be extension specific css classes in files in core.